src/shared/functions.ts

```diff
@@ -45,3 +48,51 @@ export const chunkArray = <T>(arr: T[], size: number): T[][] =>
 
 export const ensureRelative = (filePath: string, projectPath: string): string =>
   isAbsolute(filePath) ? relative(projectPath, filePath) : filePath;
+
+/**
+ * A method to help delete custom labels from a file, or the entire file if there are no more labels
+ *
+ * @param filename - a path to a custom labels file
+ * @param customLabels - an array of SourceComponents representing the custom labels to delete
+ */
+export const deleteCustomLabels = (filename: string, customLabels: SourceComponent[]): Promise<void> => {
```

Comment:
Can it return some sort of results structure?  

Like maybe it returns a Promise of either the file contents as JSON or undefined (if it was deleted) 

you'd still need the writeStub in your test, but you could assert from the function response?